student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
```

Comment:
[OPTIONAL]
To allow the user to put parent_permission while creating a person instance different than true if they want to, Kindly make sure
```suggestion
  def initialize(classroom, age, name = 'unknown', parent_permission= true)
```